packages/contracts/package.json

```
@@ -19,11 +19,11 @@
     "publish:npm": "yarn prepublish && npm publish"
   },
   "devDependencies": {
+    "@nomiclabs/hardhat-etherscan": "^3.1.0",
```

```suggestion
    "@nomiclabs/hardhat-etherscan": "3.1.0",
```

I think renovate will complain about the `^`